### ISSUE TITLE
Personal labels don't look different

### ISSUE THREAD

**Issue by Sgathec** [ZeroK-RTS/Zero-K#4021]
As far as I know, players can't tell the difference between personal labels and the kind that can be seen by other people.

**Comment by sprunk** [comment]
Technically every label is a personal label without any guarantee that anybody else can see it (for example, even with the kind that other people can see, other people might have erased theirs locally through `/clearmapmarks` -- or never even received them due to anti-draw-spam widgets).

**Comment by Arch-Shaman** [comment]
I don't think we can do anything about this without map markers being luaified?
